docs/src/convert-to-onda.md

```diff
@@ -190,17 +191,96 @@ new_plan = plan_edf_to_onda_samples_groups(new_plan)
 ### Execute the plan
 
 Once the plan has been reviewed and deemed satisfactory, execute the plan to generate `Onda.Samples` and an "executed plan" record.
-This is accomplished with the [`edf_to_onda_samples`](@ref) function, which takes an `EDF.File` and a plan as input, and returns a vector of `Onda.Samples` and the plan as executed.
-The executed plan may differ from the input plan.
-Most notably, if any errors were encountered during execution, they will be caught and the error and stacktrace will be stored as strings in the `error` field.
-It is important to review the executed plan a final time to ensure everything was converted as expected and no unexpected errors were encountered.
+This is accomplished with the [`edf_to_onda_samples`](@ref) function, which takes an `EDF.File` and a plan as input, and returns a `Vector{ConvertedSamples}`.
+For each `sensor_label` in the plan, the corresponding [`ConvertedSamples`](@ref) captures the plan rows, the resulting `Onda.Samples`, and the `sensor_label`.
+
+It is important to review the converted samples a final time to ensure everything was converted as expected and no unexpected errors were encountered.
+First, check for any converted samples with `missing` values in the `.samples` field.
+These indicated that conversion of these plan rows failed for some reason, either because the EDF signal metadata could not be matched with appropriate Onda metadata or because of a runtime error during conversion.
```

Comment:
Seems like we should mention the `error` field here:
```suggestion
These indicated that conversion of these plan rows failed for some reason, either because the EDF signal metadata could not be matched with appropriate Onda metadata or because of a runtime error during conversion. If any errors were encountered during execution, they will be caught and the error and stacktrace will be stored as strings in the `error` field.
```